docs/bot/setup.md

```diff
@@ -7,6 +7,12 @@ We do not recommend setting up an encrypted management room, and we cannot provi
 The E2EE functionality was also useful for allowing Draupnir to read encrypted messages to moderate their content in Encrypted rooms but this
 support was never reliable as its open to various exploits that are not fixable without spec changes.
 
+The nature of the exploits you can pull of if you make the choice to use E2EE in protected rooms is everything related to withholding keys.
+The spec does currently not facilitate mechanisms to allow you to get message keys from other parties so unless your draupnir gets keys from the sender
+its sol if you wanted it to be able to moderate that message. And considering the frequency of E2EE issues you couldn't sanction users for a bit of UTD automatically either.
+
+This exploit vector is considered well known to all matrix T&S professionals so this disclosure is not news to anyone in the T&S space.
```

Comment:
delete this please, we're not professionals either.

Reply:
Resolved in latest commit.